docs/source/contrib_guide_docs.rst

```diff
@@ -72,4 +72,14 @@ Questions
 Feel free to ask questions in the Google Group for Jupyter or on an open issue
 on GitHub.
 
+=========================
+Git and Github Resources
+=========================
+
+If this is your first time working with Github or git, you can leverage the following
+resources to learn about the tools.
```

Comment:
@captainsafia Let's go ahead and add the pandas link for git rebase. @JanSchulz won me over with the git fetch then git rebase combo. It's very effective and sadly underdocumented in "new to git" guides.

Also, line 49-50 in this file is missing a matching ` around the word "docs". Would you mind a quick fix? :)